wordle.py

```diff
@@ -48,7 +71,7 @@ def take_guess():
     while invalidate(guess):
         cursor_up_line_clear(2 if bad_input else 1)
         bad_input = True
-        print(invalidate(guess))
+        print(invalidate(guess) if invalidate(guess) != 'hint' else get_hint())
```

Comment:
limit `invalidate` call to once.

Reply:
Limited to twice since we feel it is more readable that way